clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10157,7 +10157,10 @@ def err_conflicting_sycl_kernel_attributes : Error<
   "conflicting attributes applied to a SYCL kernel">;
 def err_conflicting_sycl_function_attributes : Error<
    "%0 attribute conflicts with '%1' attribute">;
-
+def err_sycl_x_y_z_arguments_must_be_one : Error<
+  "%0 X-, Y- and Z- sizes must be 1 when %1 attribute is used and equals to 0">;
+def err_intel_attribute_argument_is_not_in_range: Error<
+   "%0 attribute must be in range from 0 to 3">;
```

Comment:
Can you add this range to min/max value of the attribute? And it would be nice to know why 3 is the maximum

Reply:
Currently Intel FPGA supports only max work dim of 3.